lib/generator/files/generator_cases.rb

```diff
@@ -3,16 +3,16 @@ module Files
     module GeneratorCases
       class << self
         def available(track_path)
-          cases_filepaths(track_path).map { |filepath| exercise_name(filepath) }.sort
+          cases_filepaths(track_path).map { |filepath| slugify(filepath) }.sort
         end
 
-        def class_name(exercise_name)
-          filename(exercise_name)[0..-2].split('_').map(&:capitalize).join
+        def class_name(exercise_name_or_slug)
```

Comment:
This is fine as part of the conversion process, but I **really** want to avoid ending with this.